src/include/duckdb/storage/storage_manager.hpp

```diff
@@ -46,7 +46,7 @@ class StorageManager {
 	static StorageManager &Get(Catalog &catalog);
 
 	//! Initialize a database or load an existing database from the given path
-	void Initialize();
+	void Initialize(optional_ptr<ClientContext> context);
```

Comment:
Should we also set 'nullptr' as the default value for StorageManager.Initialize?